### PR TITLE
Icon extractor

### PR DESCRIPTION
**Pull request by matandobr**
- Added 3 aapt binaries for each platform 
- Added 2 new db values, hidden and found icons
- Modified the static_analysis html to fit the icon
- Added most of the logic into a separate view - icon_analysis
- Ran tests on about 10 apks, got to any possible case -not hidden icon, hidden icon and without success guessing, and hidden icon with successful guess
Looks nice ;)

### CODE REVIEW

**Comment by ajinabraham** [comment]
Please do run the Static Analysis unittests. Android ZIP also uses the same DB model used by APK.  Just make sure that the new db entries won't break anything.

**Comment by ajinabraham** [comment]
Reviewing code.

**Comment by ajinabraham** [comment]
Code Looking Good to me.
Functionality testing now.

**Comment by ajinabraham** [comment]
@matandobr  Allow people to specify aapt binary path in settings.py.
Like 

https://github.com/MobSF/Mobile-Security-Framework-MobSF/blob/master/MobSF/utils.py#L473
and
https://github.com/MobSF/Mobile-Security-Framework-MobSF/blob/master/MobSF/settings.py#L312

**Comment by ajinabraham** [comment]
@matandobr Can you give me a source reference to this binary: https://github.com/matandobr/Mobile-Security-Framework-MobSF/blob/bd450c7a0c6961dc64475ff88fa7b2f3070c1d35/StaticAnalyzer/tools/aapt/linux/aapt

We need to be careful about shipping binaries. I had verified the mac and windows versions.

**Comment by ajinabraham** [comment]
Like I said. This breaks for android zip
```

[2017-08-01 05:48:11]
[ERROR] Saving to DB (/***/Mobile-Security-Framework-MobSF/StaticAnalyzer/views/android/db_interaction.py, LINE 203 "ICON_HIDDEN=app_dic['icon_hidden'],"): 'icon_hidden'

[2017-08-01 05:48:11]
[ERROR] Rendering to Template (/**/Mobile-Security-Framework-MobSF/StaticAnalyzer/views/android/db_interaction.py, LINE 92 "'icon_hidden': app_dic['icon_hidden'],"): 'icon_hidden'
```

**Comment by matandobr** [comment]
Made most of the fixes:
1. changed the aapt linux binary to a offical sdk one
2. added support for the zip analysis
3. minor changes 
4. unit tests - [INFO] ALL TESTS COMPLETED!
[INFO] Test Status: all tests completed
[02/Aug/2017 11:42:42] "GET /runtest/ HTTP/1.1" 200 47

File: StaticAnalyzer/views/android/icon_analysis.py [hunk 1]
```diff
@@ -73,3 +97,43 @@ def get_icon(apk_path, res_dir, tools_dir):
         }
     except:
         PrintException("[ERROR] Get icon function")
+
+
+def find_icon_path_zip(res_dir, icon_paths_from_manifest):
```
Comment (ajinabraham) [inline StaticAnalyzer/views/android/icon_analysis.py]: Does this support both Android Studio and old eclipse projects? 
Have you checked?

Comment (ajinabraham) [inline StaticAnalyzer/views/android/icon_analysis.py]: Ok I saw that you are doing

**Comment by ajinabraham** [comment]
LGTM. Please go ahead and merge to master.

**Pull request merged**